e2e/setup/config.go

```diff
@@ -73,10 +74,13 @@ func getConfig(config *Config, path string) error {
 
 // Setup used by the e2e tests. Preloaded with all necessary dependencies
 type Setup struct {
-	BridgeAccount hederaSDK.AccountID
-	SenderAccount hederaSDK.AccountID
-	TopicID       hederaSDK.TopicID
-	Clients       *clients
+	BridgeAccount   hederaSDK.AccountID
+	SenderAccount   hederaSDK.AccountID
+	TopicID         hederaSDK.TopicID
+	Clients         *clients
+	DBVerifierAlice *db_validation.Service
+	DBVerifierBob   *db_validation.Service
+	DBVerifierCarol *db_validation.Service
```

Comment:
I think that it would be best to have only 1 DB validation service that will handle the validation of the all provided databases -> `DBService`